src/examples/ui/sticky-header/css/index.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * Debounce
+ *
+ * @link https://www.javascript.christmas/2020/19
+ *
+ * @param {*} func
+ * @param {*} wait
+ * @param {*} maxWait
+ * @returns
+ */
+// function debounce(func, wait, maxWait) {
+// 	let timeout;
+// 	let max;
+//
+// 	return function () {
+// 		const functionToCall = () => {
+// 			func.apply(this, arguments);
+// 			max = null;
+// 		};
+//
+// 		if (!max) {
+// 			max = setTimeout(functionToCall, maxWait);
+// 		}
+//
+// 		if (timeout) {
+// 			clearTimeout(timeout);
+// 		}
+// 		timeout = setTimeout(functionToCall, wait);
+// 	};
+// }
```

Comment:
Yes this can be removed. OR the `debounce` and `throttle` could be moved to a more global location, like `assets/js`. and then imported. OR we could replace these functions with an npm import from `lodash` or similar

Reply:
I moved throttle to the assets directory. Removed debounce function